app/src/main/java/com/example/lapnguyen/weather_3/MainActivity.java

```diff
@@ -0,0 +1,41 @@
+package com.example.lapnguyen.weather_3;
+
+import android.content.Intent;
```

Comment:
ERROR: Unused import: android.content.Intent. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports